shared/chat/conversation/messages/attachment/image/index.js

```diff
@@ -102,7 +98,9 @@ class ImageAttachment extends React.PureComponent<Props, State> {
         >
           {!!this.props.path && (
             <ImageRender
-              ref={this.imageRef}
+              ref={ref => {
```

Comment:
why remove createRef?

Reply:
It wasn't working on native.

Reply:
Can't overwrite read-only property or something.